src/neo/SmartContract/Native/ManagementContract.cs

```diff
@@ -51,7 +51,7 @@ internal ContractState GetContract(StoreView snapshot, UInt160 hash)
         }
 
         [ContractMethod(0, CallFlags.AllowModifyStates)]
-        private ContractState Deploy(ApplicationEngine engine, byte[] nefFile, byte[] manifest)
+        internal ContractState Deploy(ApplicationEngine engine, byte[] nefFile, byte[] manifest)
```

Comment:
Why `internal`?

Reply:
It's for testing, but I can try to use a script

Reply:
Done